app/Stores/ListStore.js

```diff
@@ -80,14 +80,14 @@ class ListStore extends EventEmitter {
                     let entries = v.mapEntries(entity.name(), view.identifier(), view.getFields(), rawEntries);
 
                     // shortcut to diplay collection of entry with included referenced values
-                    v.fillReferencesValuesFromCollection(entries, view.getReferences(), true);
```

Comment:
When using `v` instead of `dataStore`, the `Datastore._entries` property is empty within the `fillReferencesValuesFromCollection` method